src/universal/components/Dashboard/DashSidebar.js

```diff
@@ -13,9 +13,9 @@ let styles = {};
 const DashSidebar = (props) => {
   const {activeArea, activeTeamId, dispatch} = props;
   const teamItems = props.user.memberships.map(m => ({
-    active: activeArea === 'team' && m.team.id === activeTeamId,
-    href: `/team/${m.team.id}`,
-    label: m.team.name
+    active: activeArea === 'team' && m.meeting.id === activeTeamId,
```

Comment:
instead of that, I'd suggest doing something like `getComponents` in the router (note the plurality). when I visit a team page, it gives me a `teamSidebar` and `teamDashboard`. Route-driven, duck-free, client side state.